sor/src/main/java/com/bazaarvoice/emodb/sor/core/AbstractCompactor.java

```diff
@@ -154,7 +155,7 @@ public Delta getDelta() {
                     if (change.getCompaction() != null) {
                         if (effectiveCompaction == null ||
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS1066)